release notes/v0.38.0.md

```diff
@@ -0,0 +1,275 @@
+    export const options = {
+      vus: 5,
+      iterations: 30,
+      thresholds: {
+          http_req_failed: ['rave<0.01'],  // "rave" is not a valid aggregation method
+      },
+    };
+    
+    ``` 
+  * thresholds applying to a non-existing metric
+    ```javascript
+    export const options = {
+      vus: 5,
+      iterations: 30,
+      thresholds: {
+          iDoNotExist: ['p(95)<200'], // The metric we apply the expression to does not exist
+      },
+    };
+    ```
+  * thresholds applying an aggregation method that's unsupported by the metric they apply to
+    ```javascript
+    export const options = {
+      vus: 5,
+      iterations: 30,
+      thresholds: {
+          my_counter: ['p(95)<200'], // Counter metrics do not support the p() aggregation method
+      },
+    };
+    ```
+
+### Support for encrypted TLS private keys
+
+It is now possible to use passphrase protected private keys when performing TLS authentication. Using the `password` property of an options' `tlsAuth` object, you can now indicate to k6 the passphrase to use to decrypt a private key. Note that this support is limited to the scope of [rfc1423](https://datatracker.ietf.org/doc/html/rfc1423) and does not support PKCS8 keys, as they're not yet supported by the golang standard library. 
+
+```javascript
+export const options = {
+  tlsAuth: [
+    {
+      domains: ['example.com'],
+      cert: open('mycert.pem'),
+      key: open('mycert-key.pem'),
+      password: 'mycert-passphrase',
+    },
+  ],
+};
+```
+
+Thanks, @Gabrielopesantos, for the [contribution](https://github.com/grafana/k6/pull/2488).
+
+### Improve JSON output's performance ([2436](https://github.com/grafana/k6/pull/2436))
+
+The JSON output was optimized and now should around 2x as more performant at outputting metrics.  This means that it either will be able to export twice as many metrics and/or to use half the resources to do the same amount of metrics. As a side effect, there is a slight breaking change in which the `tags` field is no longer sorted, while it was previously.
+
+### Treat panics as interrupt errors ([2453](https://github.com/grafana/k6/pull/2453))
+
+We changed the behavior of how k6 treats panic, which can happen during the execution of JavaScript test scripts code. Previously, the behavior was to catch the panic and log it as an error. Starting with **v0.38.0**, whenever k6 observes a panic, it logs an error like before, but more importantly, it will abort the script execution. This change should help identify and address the issues earlier.
+
+### Miscellaneous
+
+* To achieve consistency with the existing `--log-output` command-line flag, the `--logformat` flag is now considered deprecated and the new `--log-format` takes its place ([2410](https://github.com/grafana/k6/pull/2410)). The same goes for the `K6_LOGFORMAT` environment variable, which is now deprecated and replaced by `K6_LOG_FORMAT` to be consistent with the existing `K6_LOG_OUTPUT`.
```

Comment:
Isn't this a breaking change?

Reply:
It is not, as far as I understand it. When we deprecate something, we only "mark" it as deprecated, but not yet removed, hence no breaking changes in our users workflows (yet).